### PR TITLE
Bug 1365580 - Add search_rollups to the cli

### PR DESCRIPTION
**Pull request by acmiyaguchi**
Issue #90

### CODE REVIEW

**Comment by mreid-moz** [comment]
Tests are failing - can you take a look?

**Comment by acmiyaguchi** [comment]
@mreid-moz RFAL

File: mozetl/cli.py [hunk 1]
```diff
@@ -10,3 +10,4 @@ def entry_point():
 
 entry_point.add_command(churn.main, "churn")
 entry_point.add_command(dashboard.main, "search_dashboard")
+entry_point.add_command(search_rollups.main, "search_rollup")
```
Comment (mreid-moz) [inline mozetl/cli.py]: why plural in the first instance, singular in the second instance?

Comment (acmiyaguchi) [reply mozetl/cli.py]: Somewhere along the line, I forgot to grammar. The airflow job calls it `search_rollup` so this definitely calls for more consistency. The name gets used in as in "running the daily search rollup" vs "running the search rollups". I'll attach it to an upcoming patch to remove the manifest files and store the data in parquet.

**Pull request merged**